### ISSUE TITLE
The container does not work on Debian 10 (Buster)

### ISSUE THREAD

**Issue by gadget1999** [Acris/docker-shadowsocks-libev#5]
When run the container on Debian 9 (Stretch), everything works fine. But if run the same container on Debian 10 (Buster), Android client cannot connect, there is no error in the container.

I also upgraded a working Debian 9 machine to 10, then the container will stop working.

Any suggestions or tips would be appreciated.

**Comment by Acris** [comment]
Check your firewall rules first.

**Comment by Acris** [comment]
Tested on Debian 10 hosted on Digital Ocean and shadowsocks-libev operating normally.

**Comment by gadget1999** [comment]
Thanks for the quick response. On the Linux machine I upgraded from 9 to 10. It started working today (only change was re-deployed the Azure VM). On the original Debian 10 machine, it's still the same. I tried to use a web server using the same port and it's accessible (so the firewall should be fine); I also enabled debug logging on v2ray-plugin, but no logs when trying to connect.

I guess could be related to something unique for this server. Since it's not related to Debian 10, I'll close this issue.
